cylc/flow/network/resolvers.py

```diff
@@ -342,38 +343,7 @@ def __init__(self, data, **kwargs):
             result = (True, 'Command queued')
         return [{'id': w_id, 'response': result}]
 
-    async def _mutation_mapper(self, command, args):
+    async def _mutation_mapper(self, command, kwargs):
         """Map between GraphQL resolvers and internal command interface."""
-        if command in ['clear_broadcast',
-                       'expire_broadcast',
-                       'put_broadcast']:
-            return getattr(self.schd.task_events_mgr.broadcast_mgr,
-                           command, None)(**args)
-        if command == 'put_ext_trigger':
-            return self.schd.ext_trigger_queue.put((
-                args.get('event_message'),
-                args.get('event_id')))
-        if command == 'put_messages':
-            messages = args.get('messages', [])
-            for severity, message in messages:
-                self.schd.message_queue.put((
-                    args.get('task_job', None),
-                    args.get('event_time', None),
-                    severity, message))
-            return (True, 'Messages queued: %d' % len(messages))
-        if command in ['set_stop_after_clock_time',
-                       'set_stop_after_point',
-                       'set_stop_after_task']:
-            mutate_args = [command, (), {}]
-            for val in args.values():
-                mutate_args[1] = (val,)
-            return self.schd.command_queue.put(tuple(mutate_args))
-        mutate_args = [command, (), {}]
-        for key, val in args.items():
-            if isinstance(val, list):
-                mutate_args[1] = (val,)
-            elif isinstance(val, dict):
-                mutate_args[2] = val
-            else:
-                mutate_args[2][key] = val
-        return self.schd.command_queue.put(tuple(mutate_args))
+        method = getattr(self.schd.server, command)
```

Comment:
This probably needs to be a rewritten mapping instead of deletion, or an alternative put in place; like adding new command_queue associations in the Scheduler against new commands with the old ones marked for deprecation..

The reason I avoided using `self.schd.server` endpoints/methods, is because all but the `graphql` and `pb_*` endpoints will be deprecated once the CLI has migrated to the `graphql` endpoint.

This mapping is temporary scaffolding so the old (previous REST styled) endpoints can be deprecated as soon as the CLI interface switches to `graphql`, then the next step would be to remove the scaffolding and modify the internal `self.schd`/Scheduler commands as desired.

However, even though relying on `self.schd.server` methods could also be a temporary measure, it will tie up the deprecation of these endpoints with the need to change the internal commands. This really is just a different type of mapping to the internal commands.